voice/uk/ttsconfig.p

```diff
@@ -83,30 +83,30 @@ string('reached_intermediate.ogg', 'ви прибули до проміжног
 string('reached_destination.ogg','ви прибули до пункту призначення ').
 
 % NEARBY POINTS
-string('and_arrive_waypoint.ogg', 'і ви прибудете до точки GPX ').
-string('reached_waypoint.ogg', 'ви прибули до GPX ').
-string('and_arrive_favorite.ogg', 'і ви прибудете до точки улюблений ').
-string('reached_favorite.ogg', 'ви прибули до улюблений ').
-string('and_arrive_poi.ogg', ' і ви прибудете до точки інтересу ').
-string('reached_poi.ogg', 'ви прибули до точки інтересу ').
+string('and_arrive_waypoint.ogg', 'і ви прибудете до проміжної точки ').
+string('reached_waypoint.ogg', 'ви прибули до проміжної точки ').
+string('and_arrive_favorite.ogg', 'і ви прибудете до вибраної точки ').
+string('reached_favorite.ogg', 'ви прибули до вибраної точки ').
+string('and_arrive_poi.ogg', ' і ви прибудете до точки POI ').
+string('reached_poi.ogg', 'ви прибули до точки POI ').
 
 % ATTENTION
-%string('exceed_limit.ogg', 'Перевищуєте швидкість ').
+%string('exceed_limit.ogg', 'Перевищення швидкості ').
 string('exceed_limit.ogg', 'обмеження швидкості ').
 string('attention.ogg', 'Увага, ').
-string('speed_camera.ogg', 'швидкість камери').
+string('speed_camera.ogg', 'камера контролю швидкості').
 string('border_control.ogg', 'прикордонний контроль').
 string('railroad_crossing.ogg', 'залізничний переїзд').
-string('traffic_calming.ogg', 'трафіку заспокійливий').
-string('toll_booth.ogg', 'платних стенд').
+string('traffic_calming.ogg', 'трафік помірний').
```

Comment:
It was too long voice prompts, I think. On some roads in my country present series of speed bumps with low distance. It will many not useful replays.
What is logic of this prompts? If voice prompt plays only if speed more than 30 km per hour (for example), voice prompt "пристрій обмеження швидкості" is ok. Its played once and driver slow down speed.